### PR TITLE
Task/32696 integration test failures

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
- Fix sql service integration tests
- Include integration tests execution in travis

### CODE REVIEW

File: admin/tests-integration/sql.service.spec.js [hunk 1]
```diff
@@ -68,8 +68,8 @@ describe('sql.service:integration', () => {
     expect(row.version).toBeUndefined()
   })
 
-  it('dates should be stored as UTC and preserve up to 3 milliseconds', async () => {
-    const fullDateFormat = '2017-07-16T14:01:02.123+01:00'
```
Comment (GuyHarwood) [inline admin/tests-integration/sql.service.spec.js]: Typo on milliseconds

Comment (jon-shipley) [reply admin/tests-integration/sql.service.spec.js]: Those lines are being removed

File: admin/tests-integration/sql.service.spec.js [hunk 2]
```diff
@@ -68,8 +68,8 @@ describe('sql.service:integration', () => {
...
+  it('dates should be stored as UTC and preserve a less than 1000 millisecondsn difference', async () => {
```
Comment (jon-shipley) [inline admin/tests-integration/sql.service.spec.js]: Perhaps this is the typo?

File: .travis.yml [hunk 3]
```diff
@@ -51,6 +51,7 @@ install:
 # that was created in the step above.
 - docker restart mtc_funcs
 script:
+- cd ${TRAVIS_BUILD_DIR}/admin && yarn sql-integration-test
```
Comment (jon-shipley) [inline .travis.yml]: At one time the integration test rather brutally left different values in key fields like the settings table.  To be safe, we should ensure it does not alter any settings or move it to after the e2e run.

File: admin/tests-integration/sql.service.spec.js (hunk 2 above)
Comment (GuyHarwood) [reply admin/tests-integration/sql.service.spec.js]: I’m not sure 🤔 best run it through a spell checker

Comment (GeorgeChatzigiannis) [reply admin/tests-integration/sql.service.spec.js]: 😆

File: admin/tests-integration/sql.service.spec.js (hunk 1 above)
Comment (GuyHarwood) [reply admin/tests-integration/sql.service.spec.js]: good spot

**Pull request merged**